sources/packages/web/src/components/aest/students/modals/TriggerReassessmentModal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <v-form ref="reassessmentForm">
+    <modal-dialog-base :showDialog="showDialog" title="Trigger Reassement">
+      <template #content>
+        <p class="pt-1 brand-gray-text">
+          Triggering reassessment will not change student's application inputs
+          currently on file. This will rerun the student's assessment,
+          potentially requiring a school to reconfirm enrolment, and potentially
+          new eCert requests to be issued. For more information, please consult
+          the SIMS Operations Manual.
+        </p>
+        <error-summary :errors="reassessmentForm.errors" />
```

Comment:
It seems that we have been inconsistent about adding or not the `error-summary` for the modals with just the notes, but it seems that when added they are added above the content.

Reply:
Above the textarea or the text?

Reply:
As far as I could check it is supposed to be the first element inside the content. For a single note I would agree to not even have it since the others are also missing.